src/Agent.Plugins/PipelineCache/PipelineCacheTaskPluginBase.cs

```diff
@@ -1,61 +1,135 @@
 using System;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Agent.Sdk;
-using Microsoft.TeamFoundation.Build.WebApi;
 using Microsoft.TeamFoundation.DistributedTask.WebApi;
 using Microsoft.VisualStudio.Services.Agent.Util;
-using Microsoft.VisualStudio.Services.BlobStore.Common;
-using Microsoft.VisualStudio.Services.BlobStore.WebApi;
 using Microsoft.VisualStudio.Services.Common;
-using Microsoft.VisualStudio.Services.Content.Common.Tracing;
-using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.VisualStudio.Services.PipelineCache.WebApi;
 
 namespace Agent.Plugins.PipelineCache
 {
     public abstract class PipelineCacheTaskPluginBase : IAgentTaskPlugin
     {
+        protected const string RestoreStepRanVariableName = "RESTORE_STEP_RAN";
+        protected const string RestoreStepRanVariableValue = "true";
+
+        private const string SaltVariableName = "AZDEVOPS_PIPELINECACHE_SALT";
+        private const string OldKeyFormatMessage = "'key' format is changing to a single line: https://aka.ms/pipeline-caching-docs";
```

Comment:
When will the documentation be updated? I had to dig up this file to figure out what the new key format is. The linked documentation still shows the old multi-line format.

Reply:
@willsmythe  Will you be handling the new docs?